### ISSUE TITLE
download.topo() should abbreviate the filenames

### ISSUE THREAD

**Issue by dankelley** [dankelley/oce#1289]
I've been playing with a shiny app to download topo data based on mouse clicks on a coastline map, and below are some filenames I have. Sheesh. A few too many digits in the lon and lat limits, me bucko!

```
topo_63.8339980733853W_59.4546425870968W_44.162025777366N_46.7511336431949N_1min_gmt.nc
topo_64.1585230785918W_59.7328864591943W_43.6419850011772N_45.6134220231175N_1min_gmt.nc
topo_64.4057800818602W_58.9108424297981W_43.2153161640432N_45.5928762000145N_1min_gmt.nc
topo_64.426548290414W_59.24187932472W_43.2373300452276N_45.4122043161622N_1min_gmt.nc
topo_64.459965956044W_60.4625850163705W_43.0519884794111N_45.2688264151244N_1min_gmt.nc
topo_64.5354762754512W_61.0614705556911W_44.383855981029N_46.4500132589821N_1min_gmt.nc
topo_65.1781363471417W_60.9933089642478W_43.9086894739014N_46.019028346287N_1min_gmt.nc
topo_65.7008821365689W_61.3401522213493W_44.3106826304739N_46.4604601033357N_1min_gmt.nc
topo_65.955261978862W_62.7461237099622W_44.1051672018164N_45.7787757131467N_1min_gmt.nc
topo_65.9594273867077W_62.6586081193826W_46.1139804956971N_47.6832225186935N_1min_gmt.nc
topo_95.4770433326762W_94.1799328293509W_29.2455930159349N_30.1281104423815N_1min_gmt.nc
topo_95.4770433326762W_94.4899361325493W_29.4437055751113N_30.1288235783493N_1min_gmt.nc
topo_96.1213990326553W_94.4709219657629W_29.4182654364883N_30.7638669992607N_1min_gmt.nc
topo_96.6089656488588W_92.2592028915686W_28.1895675551205N_31.0612202774308N_1min_gmt.nc
```

**Comment by dankelley** [comment]
The best noaa data are at 1min resolution, or 0.01666667 of lon or lat. Therefore, I don't see any point in going past two digits after the decimal.  (The user can always specify a filename, if they want more, e.g. if they have a very detailed digital elevation model.)

**Comment by dankelley** [comment]
Fixed in `develop` branch, commit 1ba3c7c3e1b9278aed9f00c778d38f4efde6bb52
